packages/core/src/controllers/stateless/SignerCtrl.ts

```diff
@@ -1,20 +1,19 @@
 import type {
   SignerCtrlSignMessageArgs,
   SignerCtrlSignTypedDataArgs,
-  SignerCtrlWatchCallback,
-  SignerCtrlWatchOptions
+  SignerCtrlWatchCallback
 } from '../../../types/statelessCtrlTypes'
 import { ClientCtrl } from '../statefull/ClientCtrl'
 
 export const SignerCtrl = {
-  watch(options: SignerCtrlWatchOptions, callback: SignerCtrlWatchCallback) {
-    const unwatch = ClientCtrl.ethereum().watchSigner(options, callback)
+  watch(_options: undefined, callback: SignerCtrlWatchCallback) {
```

Comment:
This is fine to leave as undefined?

Reply:
It's a cost of keeping `useAsyncController` simple and as only utility hook for react (other controllers have options, callback) pattern while some only have callback